test/spec/modules/liveIntentIdSystem_spec.js

```diff
@@ -469,6 +479,13 @@ describe('LiveIntentId', function() {
     expect(result).to.eql({'lipb': {'lipbid': 'foo', 'nonId': 'foo', 'tdid': 'bar'}, 'tdid': {'id': 'bar', 'ext': {'rtiPartner': 'TDID', 'provider': provider}}});
   });
 
+  it('should decode the segments to a separate object when present', function() {
```

Comment:
It will not decode `segments` to a separate object - as discussed in the docs PR. It will be part of the `lipb` id.

Reply:
good catch, I copy pasted the text, but the expectation of test is as discussed.